File: easy_24_simple_interest_calculator/solution.cpp
#include <algorithm>
#include <iostream>
#include <iterator>
#include <sstream>
#include <string>
#include <vector>
#include <iomanip>

using namespace std;

class Solution {
public:
    double calculateSimpleInterest(double principal, double rate, double time) {
        return (principal * rate * time) / 100.0;
    }
};

void solve(istream& in = cin, ostream& out = cout) {
    double principal, rate, time;
    in >> principal >> rate >> time;
    
    Solution solution;
    double result = solution.calculateSimpleInterest(principal, rate, time);
    
    out << fixed << setprecision(2) << result << endl;
}

int main() {
    solve();
    return 0;
}

File: easy_24_simple_interest_calculator/solution.py
"""Reference implementation for the platform's official solution."""

from __future__ import annotations
import sys
from typing import TextIO

class Solution:
    def calculateSimpleInterest(self, principal, rate, time):
        """
        :type principal: float
        :type rate: float
        :type time: float
        :rtype: float
        """
        return (principal * rate * time) / 100

def solve(infile: TextIO = sys.stdin, outfile: TextIO = sys.stdout) -> None:
    line = infile.read().strip()
    principal, rate, time = map(float, line.split())
    
    solution = Solution()
    result = solution.calculateSimpleInterest(principal, rate, time)
    
    outfile.write(f"{result:.2f}\n")

if __name__ == "__main__":
    solve()
